src/java.base/share/classes/java/lang/StringLatin1.java

```diff
@@ -33,51 +33,15 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 import jdk.internal.util.ArraysSupport;
```

Comment:
```suggestion
import jdk.internal.util.ArraysSupport;
import jdk.internal.util.DecimalDigits;
```
Assuming you will restore the `getChars` here.